src/style.scss

```diff
@@ -52,9 +52,22 @@
 
 .logoIcon {
   width: 8rem;
-  border-right: 1px solid var(--auro-color-base-gray-200);
   margin-right: var(--auro-size-xs);
   padding-right: var(--auro-size-xs);
+  border-right: 1px solid transparent;
+}
+
+.logoDivider {
+  border-right-color: var(--auro-color-base-gray-200);
+}
+
+.oneworldLogo {
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  width: 2.6rem;
+  padding-left: var(--auro-size-xxs);
+  margin-bottom: .5rem;
```

Comment:
This is not a good use of css variables. Custom properties make sense when we have CSS properties that change relative to a condition in the DOM — especially a dynamic condition such as :focus, :hover, media queries or with JavaScript. 

This particularly value will not, nor should it ever be something other than the value I have tied in. Just because there happens to be a global css var with the same value does not mean we should use it in all cases.

CSS vars make sense for properties that you expect to conditionally change, such as applying different sizes to a button or to change themes.

Reply:
The change is a request to use the defined Auro Design Tokens. The primary standard of use is to define common variables between design and development. That is the role of the tokens to drive that consistency as a single source of truth.